.github/workflows/graphql.yml

```diff
@@ -0,0 +1,26 @@
+name: GraphQL API Documentation
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e
+      with:
+        ruby-version: 2.7.1
+        bundler-cache: true
+    - name: graphql-docs
+      run: |
+        cd docs
+        bundle install
+        bundle exec rake graphql:compile_docs
+    - name: Commit changes
+      uses: stefanzweifel/git-auto-commit-action@v4
+      with:
+        file_pattern: docs/
+        commit_user_name: l5io
+        commit_user_email: ci@layer5.io
+        commit_author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+        commit_options: '--signoff'
+        commit_message: "[Docs] Generated documentation for GraphQL API"

```

Comment:
We'll also need to make sure that this here commits to master

Reply:
```suggestion
        commit_message: "[Docs] Generated documentation for GraphQL API"
        branch: master
```

Reply:
@DelusionalOptimist : Sir, Anything more after `branch: master`? :>

Reply:
No @amino19. Only adding this would do it :+1: